src/helpers/voucherHelper.ts

```diff
@@ -36,7 +36,7 @@ export const storeVoucherMemberId = (memberId?: string) => {
   if (memberId) {
     addToStore(VOUCHER_MEMBER_ID, memberId);
   } else {
-    addToStore(VOUCHER_MEMBER_ID);
+    AsyncStorage.removeItem(VOUCHER_MEMBER_ID);
```

Comment:
please create the missing `removeFromStore` method in our helper and use it here.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1105/commits/36fc25dc31a63b93d2a0c9bd25c2393cafcd95ac